CONTRIBUTING.md

```diff
@@ -86,10 +95,23 @@ scripts and before building and testing your changes. More details can be found
 >        vulkaninfo
 * If you are adding or changing JSON output, please read
   [Validating vulkaninfo JSON output](https://github.com/KhronosGroup/Vulkan-Tools/blob/master/vulkaninfo/json_validation_process.md).
-  **Note:** When adding new output to vulkaninfo, do NOT add JSON output unless the formmatting is defined by a schema.
 * Run tests that explicitly exercise your changes.
 * Feel free to subject your code changes to other tests as well!
 
+#### **GitHub Cloud CI Testing**
+Pull Requests to GitHub are tested in the cloud on Linux and Windows VMs. The Linux VMs use [Travis CI](https://travis-ci.org/KhronosGroup/Vulkan-Tools) with the sequence of commands driven by the [.travis.yml](https://github.com/KhronosGroup/Vulkan-Tools/blob/master/.travis.yml) file. The Windows VMs use [AppVeyor](https://ci.appveyor.com/project/Khronoswebmaster/vulkan-tools/branch/master) with the sequence of commands driven by the [.appveyor.yml](https://github.com/KhronosGroup/Vulkan-Tools/blob/master/.appveyor.yml) file.
+
+The Linux testing includes running `vulkaninfo`.
```

Comment:
The VVL version says it runs VVL tests, which might not be a bad idea here too for mock ICD.

Reply:
@mark-lunarg @lunarpapillo Does this seem like a useful addition to CI?